variables.robot

```diff
@@ -0,0 +1,7 @@
+*** Variables ***
+
+# Variables for manual input
+${server_ip}                           192.168.4.X
```

Comment:
Why the test framework does require SSH? It should be possible to run it locally.

Reply:
Because we receive output from one connection (via SSH), and locally from the computer we send inquiries. I can't think of any other possibility to implement it differently, but I'd love to hear your idea if you have one. We can connect via ssh to ourselves, so running it locally is possible.

Reply:
SSH is no longer used